app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] OPTIONAL
It is a nice idea to use one of the Rails Active Callbacks `after_save` to call the `update_user_posts_counter` method every time a new post has been saved to database, then make the `update_user_posts_counter` method private, similar to:

```ruby
 # app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```